webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const HtmlWebPackPlugin = require( 'html-webpack-plugin' );
+const path = require( 'path' );
+module.exports = {
```

Comment:
Also configure source map for DEV. It'll become easier to debug

Reply:
used cheap-module-source-map source map in dev configuration for debugging.